### DISCUSSION TITLE
SMLIGHT SLZB-06 not really an improvement.

### DISCUSSION THREAD

**Discussion by HA-TB303** [Koenkk/zigbee2mqtt#22649] [General]
Hi, I just migrated from a SONOFF USB (CC2652P) Coördinator to a networked SMLIGHT SLZB-06.

I wrote the Sonoff's IEEE address to the SLZB-06 using ZigStarGW-MT app and migrated without issues and repairing.
Since I migrated I have devices being unavailable, slow response from time to time..... status not beging correct....etc.

Anyone having similar issues?

![CleanShot 2024-05-15 at 13 58 17](https://github.com/Koenkk/zigbee2mqtt/assets/9609805/fbf58d7f-add2-4613-9460-e8e1d8250596)

![CleanShot 2024-05-15 at 13 57 28](https://github.com/Koenkk/zigbee2mqtt/assets/9609805/ee1f5519-45b4-421b-b187-bbfff6d9b633)

**Comment by Koenkk** [comment]
The device uses the exact same chip, why did you expect a performance improvement? Nothing beats a USB coordinator 😄

**Reply by HA-TB303** [reply]
Well, I can not place the usb-coordinator in an ideal location. I also wanted to switch to a network device using the same chip so I could migrate without re-pairing. With a networked coordinator I can also make home assistant redundant.

I was expecting it to work slightly better (due to placement) or equally well, but that is clearly not the case.

There is also a lot of buzz on youtube and in forums with people praising the smlight adapter performing so well.

**Comment by TC1666** [comment]
> Well, I can not place the usb-coordinator in an ideal location. I also wanted to switch to a network device using the same chip so I could migrate without re-pairing. With a networked coordinator I can also make home assistant redundant.
> 
> I was expecting it to work slightly better (due to placement) or equally well, but that is clearly not the case.
> 

This has been my experience also. I've been running a ConBeeII USB adapter with deconz for years, I get great distance performance to my battery Aqara sensors located in a detached building. Actual direct connections.

I setup a SLZB-06 via USB connection to a RPi4 running Z2M, in the same physical location as the ConBeeII. I then switched the Aqara devices between the two different Zigbee networks, the SLZB-06 could not reliably connect to the remote devices. Both coordinators are on USB extension cables.

The SLZB-06 is a 'gimmick' in terms of that big gain antennae and higher output amplifier, what I mean by that is, what good is it that your coordinator can "shout loudly" at your remote devices if the devices themselves have less power to transmit back? That's why coordinator receiver design/sensitivity is more important than transmit power. I believe the SLZB-06 has an inferior PCB layout and is picking up a lot of noise, how else does one explain the ConBeeII with no external antennae outperforming it?

**Comment by liouma** [comment]
If you just did the migration changing the location of the coordinator, your network may need some time to reconfigure.
Now it's been a week. Did you see an improvement ?

I'm in the same situation and I just did the migration.
